pages/dkp/kommander/2.3/clusters/custom-domain-certificate/managed/index.md

```diff
@@ -0,0 +1,146 @@
+
+To customize the domain or certificate for a specific cluster after the installation of DKP, adapt or create an API `yaml` that will allow Kommander to implement the established adjustments on top of the default or any previous configuration you have made.
+
+## Configure custom domains or custom certificates
+
+<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster of the environment where you want to customize the domain or certificate by setting the <code>KUBECONFIG=<path></code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
+
+Use the API yaml to customize the domain (via the `hostname` field), the certificate (via the `issuerRef` or `certificateSecretRef` field), or both. For this, refer to the following examples:
+
+1.  Create or update and apply the `KommanderCluster` object with the wanted ingress. Remember to specify the cluster in the `kubeconfigRef` name field.
+
+    In this example, you can enter the custom domain in the `hostname` field, and an **issuer to be used by cert-manager** to issue a certificate for the domain in the `issuerRef` field.
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      issuerRef:
+        namespace: <issuer_namespace>
+        name: <issuer_name>
+    EOF
+    ```
+
+    In this example, you can enter the custom domain in the `hostname` field, and the secret in the `certificateSecretRef` field for **customized certificates created for your hostname**:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      certificateSecretRef:
+        namespace: <secret_namespace>
+        name: <secret_name>
+    EOF
+    ```
+
+<!-- 1.  TODO: Dev - Are there other steps, is there some sort of confirmation message or output?  -->
+
+## Troubleshooting
+
+If you want to ensure the customization for a domain or a certificate is completed, or if you want to obtain more information in case the customization fails, call up a list of statuses for the `KommanderCluster`:
+
+<!-- 1.  TODO: DEV - provide command -->
```

Comment:
@mhrabovcin could you provide input and output examples?

Reply:
Users should be looking on the modified `KommanderCluster` object:

```
kubect -n <workspace_namespace> get kommandercluster <cluster_name> -o yaml
```

That will print conditions that can be inspected. 

Example of a condition when ingress is still being provisioned
```
  conditions:
  - lastTransitionTime: "2022-06-24T07:48:31Z"
    message: Ingress service object was not found in the cluster
    reason: IngressServiceNotFound
    status: "False"
    type: IngressAddressReady
```

Example of state when ingress and certificate are provisioned:

```
  - lastTransitionTime: "2022-06-24T07:58:48Z"
    message: Ingress service address has been provisioned
    reason: IngressServiceAddressFound
    status: "True"
    type: IngressAddressReady
  - lastTransitionTime: "2022-06-24T07:58:50Z"
    message: Certificate is up to date and has not expired
    reason: Ready
    status: "True"
    type: IngressCertificateReady
```

To see the actual values user can look at `KommanderCluster.Status.Ingress` that looks like:

```
  ingress:
    address: 172.20.255.180
    caBundle: LS0tLS1CRUdJTiBD  ....  DQVRFLS0tLS0K
```

Reply:
Addressed in https://github.com/mesosphere/dcos-docs-site/pull/4553/commits/e4c6941f6078771b4a35480f70238368e049bad7